nameable.rb

```diff
@@ -0,0 +1,11 @@
+class Nameable
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  attr_accessor :nameable
+
```

Comment:
- [ ] I also don't see the `CapitalizeDecorator` class. 🤔 kindly, Create a class that inherits from the base Decorator class. and make sure that you Implement a method correct_name that capitalizes the output of @nameable.correct_name.

- The `CapitalizeDecorator` class, might look similar to:

```ruby
# Inherits from Decorator
class CapitalizeDecorator < Decorator
    # Capitalize the first letter of the given name.
    def correct_name
      @nameable.correct_name.capitalize
    end
end
```

- [ ] I also don't see the `TrimmerDecorator` class. 🤔 kindly, Create a class that inherits from the base Decorator class. Implement a method correct_name that makes sure that the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.

- The  `TrimmerDecorator` class, might look similar to:

```ruby
# Inherits from Decorator
class TrimmerDecorator < Decorator
    # The name has a maximum of 10 characters. If it's longer it should trim the word.
    def correct_name
       ....
    end
end
```

Reply:
Hi thank you the review :)
All the decorator classes are inside the `base.rb` file.

Reply:
Hi @iBel251, I can now see that you implemented the  `CapitalizeDecorator` class and the `TrimmerDecorator` class inside the  `base.rb` file. Great job 👏🏻  however, your code does not work as expected see the screenshot 👇🏻 


![033](https://user-images.githubusercontent.com/79058364/182581468-8f3ffed2-6380-4f10-b38b-2f58ad021bfa.png)

Reply:
oww yes i see it now, I used `capitalize` instead of `upcase` keyword. I'll fix it now.